doc/source/law-and-order-special-victims-unit.md

```diff
@@ -0,0 +1,31 @@
+# ⚖ Law and order Special Victims Unit
+
+Law and Order was a series created in the 1980s by Dick Wolf and co-produced with Machael Mann. It was so successful, that they never imagined that the series would continue to this day, the series begins when the rookie Olivia Benson begins to work in a police force called the "Special Victims Unit" that is dedicated to investigating sexual crimes in the city of New York.
+
+![Law And Order](_static/images/law-and-order-special-victims-unit/law-and-order.jpg)
+
+Since its inception, the series has dealt with complex cases that were rarely seen on television, such as: domestic violence, rape, serial murder, abuse, hate crimes, among others. Of course, in most cases, the victims of this type of abuse are usually women and in the background also homosexuals, lesbians and children.
+
+The investigations are based on real news that have managed to impact society and it is possible that the great success of the series comes from there. The universality and topicality of its themes immediately catch everyone's attention, since physical or psychological aggression never ceases to move the television audience. Actress Mariska Hargitay has earned the respect of viewers over the years, although it is true that her character began as secondary and supporting detectives Elliot Stabler starring Chistopher Meloni and detective John Munch (Richard Belzer) . Olivia Benson was violent and impulsive, but gradually changed as the series progressed and became the most intelligent and mature person in the cast, probably because of the closeness with which the actress treated the cases and in some way, this interpreter It was becoming a symbol of the fight for equality, to the point that he received letters from people in gratitude for making them aware of their reality and that in many cases they did not want to acknowledge. Thanks to the messages received from her fans, the concern of helping in a real way was generated in the actress and that is why she created an association to support victims of sexual crimes in her country.
+
+This American television series premiered on September 20, 1999, is produced and set in New York City where a whole range of police, procedural, legal and criminal drama is reflected. The media have contributed their headlines to the script of the series, since the most notorious sexual crimes have been taken into account for the realization of the scripts of law and order.
```

Comment:
```suggestion
This American television series premiered on September 20, 1999, is produced and set in New York City where a whole range of police, procedural, legal and criminal drama is reflected. The media have contributed their headlines to the script of the series, since the most notorious sexual crimes have been taken into account for the realization of the scripts of Law & Order.
```